docs/content/getting_started/shopping_list_app.md

```diff
@@ -708,7 +708,7 @@ We are now ready to deploy our API. To do so, run the following steps:
 ```bash
 pdk build
 cd packages/infra
-pdk run deploy --require-approval never
+pdk deploy:dev
```

Comment:
does this work with pnpm?

Reply:
Yep :) `pdk deploy:dev` works but `pdk deploy` doesn't - have updated the script so `pdk deploy` works for pnpm too :)